static/global.css

```diff
@@ -1,21 +1,62 @@
+/*
 @font-face {
-  font-family: 'Open Sans Light';
-  src: url('/font/OpenSans-Light.ttf') format('truetype');
+	font-family: 'Open Sans Light';
+	src: url('/font/OpenSans-Light.ttf') format('truetype');
 }
 @font-face {
-  font-family: 'Open Sans Regular';
-  src: url('/font/OpenSans-Regular.ttf') format('truetype');
+	font-family: 'Open Sans Regular';
+	src: url('/font/OpenSans-Regular.ttf') format('truetype');
 }
 @font-face {
-  font-family: 'Open Sans Semibold';
-  src: url('/font/OpenSans-Semibold.ttf') format('truetype');
+	font-family: 'Open Sans Semibold';
+	src: url('/font/OpenSans-Semibold.ttf') format('truetype');
+}
+@font-face {
+    font-family: 'clfallback';
+    src: url('/font/OpenDyslexic3-Regular.otf') format('opentype')
+}
+*/
+@font-face {
+    font-family: 'Open Dyslexia';
+    src: url('/font/OpenDyslexic-Regular.otf') format('opentype');
+    font-weight: 400;
+    font-style: normal
+}
+@font-face {
+    font-family: 'Open Dyslexia';
+    src: url('/font/OpenDyslexic-Italic.otf') format('opentype');
+    font-weight: 400;
+    font-style: italic
+}
+@font-face {
+    font-family: 'Open Dyslexia';
+    src: url('/font/OpenDyslexic-Bold.otf') format('opentype');
+    font-weight: 700;
+    font-style: normal
+}
+@font-face {
+    font-family: 'Open Dyslexia';
+    src: url('/font/OpenDyslexic-Bold-Italic.otf') format('opentype');
+    font-weight: 700;
+    font-style: italic
+}
+@font-face {
+    font-family: 'Archivo';
+    src: url('/font/Archivo-VariableFont_wdth,wght.ttf') format('truetype');
+    font-weight: 400;
+    font-style: normal
+}
+@font-face {
+    font-family: 'Archivo';
+    src: url('/font/Archivo-Italic-VariableFont_wdth,wght.ttf') format('truetype');
+    font-weight: 400;
+    font-style: italic
 }
 
 * {
 	box-sizing: border-box;
 	margin: 0;
 	padding: 0;
-	font-family: 'Open Sans Light', sans-serif;
```

Comment:
Just a note that as we discussed we need to restore some fallback, at least `sans-serif`.